### PR TITLE
St slides

### LINKED ISSUE #99: add slides from September C&C
delayed upload of slides on fucntions from September's C&C

### PR DESCRIPTION
**Pull request by sallyt-nhs**
closes #99

### CODE REVIEW

File: presentations/2023-09-07_coffee_and_coding_functions/index.qmd [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+library(purrr)
+library(lubridate)
+library(zoo)
+library(dplyr)
+library(janitor)
+library(ggpubr)
+library(StrategyUnitTheme)
+
+# drive loc
+zdrive_loc <- "Z:/Strategic Analytics/Projects 2022/1056 WMAP Cardiac/SD Modelling/"
```
Comment (matt-dray) [inline presentations/2023-09-07_coffee_and_coding_functions/index.qmd:39]: Aha, I don't have access to this folder so I can't rebuild the slides. Would it be possible to create a small dataframe within this code chunk, or perhaps read in a small demo csv that you add to the `presentations/2023-09-07_coffee_and_coding_functions/` folder?

**Review by matt-dray — CHANGES_REQUESTED** [review]
Hello, thank you for the submission! One problem: I can't rebuild the slides (see the line comment).

File: presentations/2023-09-07_coffee_and_coding_functions/index.qmd (hunk 1 above)
Comment (sallyt-nhs) [reply presentations/2023-09-07_coffee_and_coding_functions/index.qmd]: csv file added and the code altered

**Review by matt-dray — APPROVED** [review]
Thank you, that runs now. Congratulations on your PR being #100 for the repo! 💯

**Pull request merged**